tests/openvino/utils_tests.py

```diff
@@ -115,13 +116,19 @@
 }
 
 
+_ARCHITECTURES_TO_EXPECTED_INT4_INT8 = {"opt125m": (82, 323)}
```

Comment:
what part of the model will be quantized to int8 ? Does this target specific parameters (who could be sensitive to quantization) ?

Reply:
all the matmuls with weights. By default its 80% in int4 and the rest in INT8.

Reply:
very clear, thanks!